gremlin-go/driver/resultSet.go

```diff
@@ -148,7 +148,7 @@ func (channelResultSet *channelResultSet) Channel() chan *Result {
 	return channelResultSet.channel
 }
 
-func (channelResultSet *channelResultSet) one() (*Result, error) {
+func (channelResultSet *channelResultSet) One() (*Result, error) {
```

Comment:
Do we need documentation if we are exporting?

Reply:
I believe so. Here is a suggestion based on the Javadoc. 
```suggestion
// One returns the next Result from the channelResultSet, blocking until one is available.
func (channelResultSet *channelResultSet) One() (*Result, error) {
```